src/app/index.tsx

```diff
@@ -0,0 +1,4 @@
+import { withProviders } from "./providers";
+import { RowApp } from "./ui";
+
+export const App = withProviders(RowApp);
```

Comment:
RowApp??? Did you mean Raw?

Reply:
possibly it was the thing i thought about but didnt write

Reply:
changed